tilauskasittely/editilaus_in.inc

```diff
@@ -2384,7 +2384,7 @@ while ($tietue = fgets($fd)) {
         // Vientitilauksille saa kutsua tilaus-valmis.inci� vaan kerran
         // Kotimaan tilauksilel kutsutaan tilaus-valmis.inci� kahdesti,
         // koska ekalla kerralla niille luodaan pupeen lasku
-        if ($laskurow["vienti"] == "") {
+        if (($laskurow["vienti"] == "" or $edi_tyyppi == "" ) and $laskurow['h1time'] == '0000-00-00 00:00:00') {
```

Comment:
Kyllä. Mun kysymys oli, että miksi olemme valinneet juuri noi ehdot?

> On 20 Apr 2016, at 10:02, satuaalto <notifications@github.com> wrote:
> 
> In tilauskasittely/editilaus_in.inc <https://github.com/devlab-oy/pupesoft/pull/2988#discussion_r60358350>:
> 
> > @@ -2384,7 +2384,7 @@ while ($tietue = fgets($fd)) {
> >          // Vientitilauksille saa kutsua tilaus-valmis.inci� vaan kerran
> >          // Kotimaan tilauksilel kutsutaan tilaus-valmis.inci� kahdesti,
> >          // koska ekalla kerralla niille luodaan pupeen lasku
> > -        if ($laskurow["vienti"] == "") {
> > +        if (($laskurow["vienti"] == "" or $edi_tyyppi == "" ) and $laskurow['h1time'] == '0000-00-00 00:00:00') {
> h1time päivittyy, kun käydään tilaus-valmis.inc:ssä ja edi_tyyppi on tyhjä silloin, kun valittuna on tilaustyyppi 'Editilaus'
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/devlab-oy/pupesoft/pull/2988/files/5300ef6b4fea346e2afb2412d8baf0071a40da25#r60358350>